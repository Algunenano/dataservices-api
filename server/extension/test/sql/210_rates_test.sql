SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = 'cdb_service_get_rate_limit'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text');

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = 'cdb_service_set_user_rate_limit'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, json');

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = 'cdb_service_set_org_rate_limit'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, json');

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = 'cdb_service_set_server_rate_limit'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, json');
