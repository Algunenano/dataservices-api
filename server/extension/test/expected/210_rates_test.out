SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = 'cdb_service_get_rate_limit'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text');
 exists 
--------
 t
(1 row)

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = 'cdb_service_set_user_rate_limit'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, json');
 exists 
--------
 t
(1 row)

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = 'cdb_service_set_org_rate_limit'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, json');
 exists 
--------
 t
(1 row)

SELECT exists(SELECT *
              FROM pg_proc p
              INNER JOIN pg_namespace ns ON (p.pronamespace = ns.oid)
              WHERE ns.nspname = 'cdb_dataservices_server'
              AND proname = 'cdb_service_set_server_rate_limit'
              AND oidvectortypes(p.proargtypes)  = 'text, text, text, json');
 exists 
--------
 t
(1 row)

